integration_tests/__tests__/start.test.js

```diff
@@ -20,9 +20,13 @@ const TEST_PROJECT_DIR = path.resolve(
 );
 
 let haul;
-beforeAll(done => {
+beforeAll(async done => {
   run('yarn --mutex network', TEST_PROJECT_DIR);
-  haul = runHaul(TEST_PROJECT_DIR, ['start']);
+  try {
+    haul = await runHaul(TEST_PROJECT_DIR, ['start']);
+  } catch (error) {
+    done.fail(error);
+  }
```

Comment:
you can get rid of the `try/catch` and `done` entirely, error will be caught normally